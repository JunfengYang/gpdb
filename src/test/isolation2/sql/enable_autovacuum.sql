!\retcode gpconfig -c autovacuum -v on --skipvalidation;
!\retcode gpstop -ari;
! psql postgres -c "show autovacuum;";
