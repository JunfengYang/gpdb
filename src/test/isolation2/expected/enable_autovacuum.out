!\retcode gpconfig -c autovacuum -v on --skipvalidation;
(exited with code 0)
!\retcode gpstop -ari;
(exited with code 0)
! psql postgres -c "show autovacuum;";
 autovacuum 
------------
 on
(1 row)


